vite.config.ts

```diff
@@ -9,12 +10,17 @@ import cesium from "vite-plugin-cesium";
 import dts from "vite-plugin-dts";
 import svgr from "vite-plugin-svgr";
 import { configDefaults } from "vitest/config";
+
+const cesiumPackageJson = JSON.parse(
+  readFileSync(resolve(__dirname, "node_modules", "cesium", "package.json"), "utf-8"),
+);
+
 // https://vitejs.dev/config/
 export default defineConfig(() => ({
   plugins: [
     svgr(),
     react(),
-    cesium({ rebuildCesium: true }),
+    cesium({ rebuildCesium: true, cesiumBaseUrl: `cesium-${cesiumPackageJson.version}/` }),
```

Comment:
```suggestion
```
I think we don't need use this plugin in here, because Cesium is specified as external in [here](https://github.com/reearth/core/pull/43/files#diff-6a3b01ba97829c9566ef2d8dc466ffcffb4bdac08706d3d6319e42e0aa6890ddR37). And user should use this setting in user's project.

Reply:
Updated! Thank you!

Reply:
I think you don't need to load this plugin 👀

Reply:
Removed! Thank you!